lib/blockscore/actions/delete.rb

```diff
@@ -0,0 +1,9 @@
+module BlockScore
+  module Actions
+    module Delete
+      def delete(id, options)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.